tools/build-for-arm.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
We should probably have some notice that this is experimental only, etc?

Reply:
Yes, 1 note would be added here.